docs-core/src/main/resources/db/update/dbupdate-021-0.sql

```diff
@@ -1,4 +1,4 @@
 alter table T_DOCUMENT add column DOC_IDFILE_C varchar(36);
-alter table T_DOCUMENT add constraint FK_DOC_IDFILE_C foreign key (DOC_IDFILE_C) references T_FILE (FIL_ID_C) on delete restrict on update restrict;
+-- alter table T_DOCUMENT add constraint FK_DOC_IDFILE_C foreign key (DOC_IDFILE_C) references T_FILE (FIL_ID_C) on delete restrict on update restrict;
```

Comment:
Take a look at these changes in the database because we need to be careful with removing already existing tables/fields